sunpy/physics/differential_rotation.py

```diff
@@ -153,10 +150,15 @@ def rot_hpc(x, y, tstart, tend, frame_time='synodic', rot_type='howard', **kwarg
     by convert_hpc_hg.  This leads to very slightly different results from
     rot_hpc compared to rot_xy.
     """
+    x = coord.Tx
+    y = coord.Ty
 
+    '''
+    # Is this necessary?
```

Comment:
SkyCoord supports creation of coordinate object even if the shape of arrays aren't same.